Gemfile.lock

```diff
@@ -32,7 +32,7 @@ GEM
     celluloid (0.16.0)
       timers (~> 4.0.0)
     childprocess (3.0.0)
-    concurrent-ruby (1.1.6)
+    concurrent-ruby (1.1.7)
```

Comment:
which command did you run to bump the gem?

```
bundle update zendesk_apps_support  --conservative
```
or 
```
bundle update zendesk_apps_support 
```

Reply:
I just changed the gem by hand, then ran `bundle install`.

I'll try `bundle update zendesk_apps_support  --conservative` and see if fewer versions are changed.

Note though that we do expect some gem versions to change, because of this pr being merged into ZAS here: https://github.com/zendesk/zendesk_apps_support/pull/259

Reply:
:) No problem, I remember `bundle update zendesk_apps_support  --conservative` will not allow shared dependencies to be updated. Just give a try and see whether you will have any difference or not.